src/js/controllers/ServicesCtrl.js

```diff
@@ -232,6 +234,27 @@ function ServicesListCtrl($scope, $location, $route, $routeParams, profileServic
       });
     }
   });
+
+  $scope.subscribeToNewsLetter = function(){
+      $scope.query.status = true;
+      $scope.servicesPromise = profileService.getServices($scope.query).then(function (response) {
+        if (response.status == 200) {
+          response.data.forEach(function (service) {
+            if(service.name == 'Humanitarian ID News'){
```

Comment:
Rely on ID rather than name. If at some point someone decides to change the name of the HID news service, this code will fail. The ID will not change however, so rely on the ID that's in production.